run.py.in

```diff
@@ -11,8 +11,12 @@ which satisifies that requirement.
 """
 
 if __name__ == '__main__@SOME_UNDEFINED_VARIABLE_CMAKE_WILL_REMOVE@':
+    # This is stupid but without it, Cython/Python cannot find the openage module.
+    import sys
+    sys.path.append('./bin')
```

Comment:
you have to either run it in bin/ or add 
this path dynamically if we detect we're in the build dir currently- it shouldn't be generally added since this may be the installed openage binary

Reply:
I did run it in `bin` but it failed regardless. Don't know why but it worked before for earlier cython versions.

Reply:
hm well, but we shpuld try a proper fix (which may include adding py paths of course)

Reply:
changed it to `sys.path.append(os.getcwd())`.